operation/fhir-operation-bulkdata/src/main/resources/hl7/fhir/uv/bulkdata/package/CapabilityStatement-bulk-data.json

```diff
@@ -0,0 +1 @@
s implementing this capability statement should meet the requirements set by the [Bulk Data Implementation Guide](https://fhir.org/ig/HL7/bulk-data/). FHIR Bulk Data Clients have the option of choosing from this list to access necessary data based on their local use cases and other contextual requirements.","jurisdiction":[{"coding":[{"system":"http://unstats.un.org/unsd/methods/m49/m49.htm","code":"001","display":"World"}]}],"kind":"requirements","fhirVersion":"4.0.0","format":["json"],"patchFormat":["application/json-patch+json"],"implementationGuide":["http://hl7.org/fhir/uv/bulkdata/ImplementationGuide/ig"],"rest":[{"mode":"server","documentation":"The Bulk Data (Flat FHIR) Server **SHOULD**:\n\n1. Export data from a FHIR server whether or not it is associated with a patient. This supports use cases like backing up a server or exporting terminology data by restricting the resources returned using the _type parameter.\n1. Obtain data on all patients listed in a single FHIR Group Resource. If a FHIR server supports Group-level data export, it SHOULD support reading and searching for Group resource. This enables clients to discover available groups based on stable characteristics such as Group.identifier. Note: How these groups are defined is implementation specific for each FHIR system. For example, a payer may send a healthcare institution a roster file that can be imported into their EHR to create or update a FHIR group. Group membership could be based upon explicit attributes of the patient, such as: age, sex or a particular condition such as PTSD or Chronic Opioid use, or on more complex attributes, such as a recent inpatient discharge or membership in the population used to calculate a quality measure. FHIR-based group management is out of scope for the current version of this implementation guide.\n1. Export data from a FHIR server for all data associated with patients. This supports use cases like transmitting all data about patients or clinical care between systems.","security":{"description":"1. See the [Authorization Guide](authorization/index.html) section for requirements and recommendations.\n1. This specification describes requirements for requesting an access token\n        through the use of an OAuth 2.0 client credentials flow, with a [JWT\n        assertion](https://tools.ietf.org/html/rfc7523) as the\n        client's authentication mechanism."},"operation":[{"extension":[{"url":"http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation","valueCode":"SHOULD"}],"name":"export","definition":"OperationDefinition/BulkDataExport"},{"extension":[{"url":"http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation","valueCode":"SHOULD"}],"name":"patient-export","definition":"OperationDefinition/PatientLevelExport"},{"extension":[{"url":"http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation","valueCode":"SHOULD"}],"name":"group-export","definition":"OperationDefinition/GroupLevelExport"}]}]}
```

Comment:
Note, we also support import, which is not listed here, and we've jammed into the capability statement.

Reply:
Are you suggesting an update to the capability statement?  Only its URL is externalized via the metadata operation.

Reply:
I'm just warning that we have one more operation in case we need to create a custom capability statement for this case.

Reply:
I think $import is at the System level and Bulk Status is at the System Level, I think this doesn'timpact it.  Right?

Reply:
Right, the bulkdata Capability Statement only mentions export.  Import is additional functionality not mentioned by the bulkdata Capability Statement.

Reply:
Right - to clarify, I'm just making sure that it is still listed in the `metadata`

Reply:
Yes, this does not change the values returned in the CapabilityStatement->rest->operation field, which is where the import operation is described.

Reply:
great 👍